src/Features/Core/Portable/InlineHints/TypeHint.cs

```diff
@@ -14,24 +14,27 @@ namespace Microsoft.CodeAnalysis.InlineHints
 
         public ITypeSymbol Type { get; }
         public TextSpan Span { get; }
+        public TextChange? TextChange { get; }
         public ImmutableArray<SymbolDisplayPart> Prefix { get; }
         public ImmutableArray<SymbolDisplayPart> Suffix { get; }
 
-        public TypeHint(ITypeSymbol type, TextSpan span, bool leadingSpace = false, bool trailingSpace = false)
+        public TypeHint(ITypeSymbol type, TextSpan span, TextChange? textChange, bool leadingSpace = false, bool trailingSpace = false)
```

Comment:
when is textChange passed null here?

Reply:
Lambda parameter types and SingleVariableDesignationSyntax